src/Serialization/main.jl

```diff
@@ -1,6 +1,14 @@
 using JSON
 using UUIDs
 
+OscarBasicType = Union{
+    fmpz,
+    fmpq,
+    Bool,
+    Number,
+    Symbol
```

Comment:
To reduce future diffs, use a trailing comma?
```suggestion
    Symbol,
```